physics/src/modules/include/IIceAlbedo.hpp

```diff
@@ -21,6 +23,12 @@ class IIceAlbedo {
      * @param snowThickness The true snow thickness on top of the ice.
      */
     virtual double albedo(double temperature, double snowThickness) = 0;
+
+    /*!
+     * Sets the time parameter for the implementation, if it is time dependent.
+     * @param time The desired TimePoint.
+     */
+    virtual void setTime(const TimePoint& tp) { }
```

Comment:
Wouldn't something along those lines make sense?

```suggestion
    virtual void setTime(const TimePoint& tp) { M_tp = tp; }
    
    private:
        TimePoint M_tp;
```

Reply:
That's a detail for each implementation to solve. None of the existing implementations are time dependent, so they inherit the do-nothing Base class version of the function.

Reply:
Fair enough!